inc/functions_post.php

```diff
@@ -1067,3 +1067,28 @@ function return_bytes($val) {
 
 	return intval($val);
 }
+
+/**
+ * Detects whether an attachment removal/approval/unapproval
+ * submit button was pressed (without triggering an AJAX request)
+ * and sets inputs accordingly (as for an AJAX request).
+ */
+function detect_attachmentact() {
+	global $mybb;
+
+	foreach ($mybb->input as $key => $val) {
```

Comment:
Nitpick: this should follow our code style:

```suggestion
	foreach($mybb->input as $key => $val)
	{
```